src/lib/models/useCheckMaxLengthFromSchema.ts

```diff
@@ -1,14 +1,20 @@
 import { createMaxCharacterLength } from '@dhis2/ui'
 import { useMemo } from 'react'
-import { SchemaName } from '../../types'
+import { SchemaFieldProperty, SchemaName } from '../../types'
 import { useSchema } from '../schemas'
 
 export function useCheckMaxLengthFromSchema(
```

Comment:
I think we can simplify/clean this up by this by passing `schema` instead of `schemaName` - and then make it a regular function instead of a hook.

Reply:
That was my intention by introducing checkMaxLengthFromProperty, with the idea of moving to this new version as we move the validators to the more generic validator we created. hat do you think?